docs/tutorials/hello-world-application/creating-the-database.mdx

```diff
@@ -27,27 +38,53 @@ In the Kuneiform IDE, you must connect to the endpoint for the Kwil Network you
 
 Next, add the network details in the popup Menu. After you click "OK", the IDE will automatically switch to the new network.
 
+> **TODO** Believe that https://provider.kwil.com would make sense for a 1st pass Tutorial example as it does NOT require the user to set up their own network as a prerequisite.
+
+> **TODO** How can I delete or Edit a Workspace once I've added it in the Kuneiform IDE?
```

Comment:
You can right click over the workspace and click delete. Is this something that should be included in the tutorial?